### DISCUSSION TITLE
Release of Kakao provider

### DISCUSSION THREAD

**Discussion by 26filip** [SocialiteProviders/Providers#1190] [General]
Hi,

it's been a while since last release of [Kakao](https://github.com/SocialiteProviders/Kakao) provider. It'll be very useful to be able to use next release specially because of fix in commit [d22dd4d](https://github.com/SocialiteProviders/Kakao/commit/d22dd4de45c2e891a7c36baa60f7d093a350bacb).

Thanks in advance

**Comment by atymic** [comment]
Tagged :)
